gwcli/ceph.py

```diff
@@ -288,11 +288,18 @@ class RadosPool(UINode):
     def __init__(self, parent, pool_name, pool_md):
         UINode.__init__(self, pool_name, parent)
         self.pool_md = pool_md
+        k = 'N'
+        m = 'N'
+        if pool_md['type'] == 3:
+            result, err = run_shell_cmd(
```

Comment:
I think you renamed `_run_ceph_cmd` to `run_shell_cmd` in your other PR.  Since this is against master, need to use what is in master.

Reply:
Yeah, right. I test this together with https://github.com/ceph/ceph-iscsi/pull/237 and separate it here.